### ISSUE TITLE
Hovering over the sidebar, `.app-name` moves 5px

### ISSUE THREAD

**Issue by dunhamjared** [jhildenbiddle/docsify-themeable#22]
When you hover over the sidebar, `.app-name` moves 5px to the left.

Had to apply a custom css style for this not to happen:

```css
body:not([data-platform^="Mac"]) .sidebar:hover>*:not(.sidebar-nav){
    margin-right: 0;
}
```

Chrome Version 73.0.3683.103 (Official Build) (64-bit)
Using: https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/css/theme-simple-dark.css

**Comment by itsmingjie** [comment]
Ran into this as well and couldn't find out where it came from — thanks @dunhamjared for the fix!

**Comment by jhildenbiddle** [comment]
Thanks for filing the issue, @dunhamjared.  

This has been fixed in v0.7.2. If you're loading docsify-themeable using a version tag in the url (i.e. `@0` or `@latest`) you may need to wait for the CDN redirects to update.

The 5px shift you were seeing was an attempt I made to emulate macOS's "hide scrollbars until hovered" behavior. The [docsify.js site](https://docsify.js.org/#/configuration) handles this by rending the scrollbar but making it transparent until hovered. This works, but it means the sidebar navigation selection (the green border) is removed from the sidebar border when the sidebar content is scrollable.

Hover off:

![hover_off](https://user-images.githubusercontent.com/442527/56097488-cccd4780-5ea9-11e9-95c9-be5099ea8ae9.png)

Hover on:

![hover_on](https://user-images.githubusercontent.com/442527/56097490-d3f45580-5ea9-11e9-8f52-a05cbadcf727.png)

I attempted to address this by setting the border width to `0px` by default, then increasing the width on hover (instead of adjusting the scrollbars transparency). This works, but it causes the `.app-name` element to shift so I added a rule to "unshift" it on hover. Unfortunately, this fix only works when the sidebar is scrollable. If it isn't, then the rule I added causes the shift that you saw. The only way to fix this would be via JavaScript, which seems unnecessary. So, I simply removed the "hide until hover" effect. The end result is that Chrome (or all webkit browsers) now display scrollbars the same way Edge and Firefox do on Windows.
